lib/log.js

```diff
@@ -66,17 +66,22 @@ Lug.prototype.event = function (name, data) {
   process.stdout.write(JSON.stringify(e) + '\n')
 }
 
-Lug.prototype.activityEvent = function(event, uid, request) {
+Lug.prototype.activityEvent = function(event, request, data) {
   var info = {
-    event: event,
-    uid: uid,
```

Comment:
Logging an error sounds good to me, will update and merge along with https://github.com/mozilla/fxa-auth-db-mysql/pull/105.